### PR TITLE
Code and documentation improvements

### PR DESCRIPTION
**Pull request by ITALIA195**
Improves the code readability and tries to keep a regular codestyle within the multiple projects.
The PR contains drastic changes but the tutorials will be much more readable and maintenance will be easier.

### CODE REVIEW

File: Chapter 1/1 - Creating a Window/Window.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace LearnOpenGL_TK
 {
     // This is where all OpenGL code will be written.
     // OpenTK allows for several functions to be overriden to extend functionality; this is how we'll be writing code.
-    internal class Window : GameWindow
+    public class Window : GameWindow
```
Comment (ITALIA195) [inline Chapter 1/1 - Creating a Window/Window.cs]: Although making all classes internal isn't that bad I didn't mean to do it in d2da622674aef1cbe6671316549dd3ece771c19e.
The internal modifier was added by rider; I'm now removing it.

File: Chapter 1/1 - Creating a Window/Program.cs [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
-                //To create a new window, create a class that extends GameWindow, then call Run() on it.
-                //Run takes a double, which is how many frames per second it should strive to reach.
-                //You can leave that out and it'll just update as fast as the hardware will allow it.
-                game.Run(60.0);
-            }
+            // To create a new window, create a class that extends GameWindow, then call Run() on it.
+            // Run takes a double, which is how many frames per second it should strive to reach.
+            // You can leave that out and it'll just update as fast as the hardware will allow it.
+            using (var window = new Window(800, 600, "LearnOpenTK - Creating a Window"))
+                window.Run(60.0);
```
Comment (Perksey) [inline Chapter 1/1 - Creating a Window/Program.cs]: Revert this, it's against the style guide.

File: Chapter 1/2 - Hello Triangle/Program.cs [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            using (var game = new Game(800, 600, "LearnOpenTK - Hello Triangle!"))
-            {
-                game.Run(60.0);
-            }
+            using (var window = new Window(800, 600, "LearnOpenTK - Hello Triangle!"))
+                window.Run(60.0);
```
Comment (Perksey) [inline Chapter 1/2 - Hello Triangle/Program.cs]: You must use braces, revert.

File: Chapter 1/3 - Element Buffer Objects/Program.cs [hunk 4]
```diff
@@ -1,13 +1,11 @@
...
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            using (var game = new Game(800, 600, "LearnOpenTK - Element Buffer Objects"))
-            {
-                game.Run(60.0);
-            }
+            using (var window = new Window(800, 600, "LearnOpenTK - Element Buffer Objects"))
+                window.Run(60.0);
```
Comment (Perksey) [inline Chapter 1/3 - Element Buffer Objects/Program.cs]: Revert.

File: Chapter 1/4 - Textures/Program.cs [hunk 5]
```diff
@@ -1,13 +1,11 @@
...
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            using (var game = new Game(800, 600, "LearnOpenTK - Textures"))
-            {
-                game.Run(60.0);
-            }
+            using (var window = new Window(800, 600, "LearnOpenTK - Textures"))
+                window.Run(60.0);
```
Comment (Perksey) [inline Chapter 1/4 - Textures/Program.cs]: Revert

File: Chapter 1/4 - Textures/Shaders/shader.frag [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+// A sampler2d is the representation of a texture in a shader.
+// Each sampler is bound to a texture unit (texture units are described in Texture.cs on the Use function)
+// By default, the unit is 0, so no code-related setup is actually needed.
+// Multiple samplers will be demonstrated in section 1.5
+uniform sampler2D texture0;
+
+void main()
+{
+	// To use a texture, you call the texture() function.
+	// It takes two parameters: the sampler to use, and a vec2, used as texture coordinates
```
Comment (Perksey) [inline Chapter 1/4 - Textures/Shaders/shader.frag]: Padding looks a little off here.

File: Chapter 1/4 - Textures/Shaders/shader.vert [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+// ...However, they aren't needed for the vertex shader itself.
+// Instead, we create an output variable so we can send that data to the fragment shader.
+
+out vec2 texCoord;
+
+void main(void)
+{
+	// Then, we further the input texture coordinate to the output one.
+	// texCoord can now be used in the fragment shader.
```
Comment (Perksey) [inline Chapter 1/4 - Textures/Shaders/shader.vert]: Revise padding.

File: Chapter 1/5 - Multiple Textures/Program.cs [hunk 8]
```diff
@@ -1,13 +1,11 @@
...
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            using (Game game = new Game(800, 600, "LearnOpenTK - Multiple Textures"))
-            {
-                game.Run(60.0);
-            }
+            using (var window = new Window(800, 600, "LearnOpenTK - Multiple Textures"))
+                window.Run(60.0);
```
Comment (Perksey) [inline Chapter 1/5 - Multiple Textures/Program.cs]: Revert

File: Chapter 1/6 - Transformations/Program.cs [hunk 9]
```diff
@@ -1,13 +1,11 @@
...
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            using (Game game = new Game(800, 600, "LearnOpenTK - Transformations"))
-            {
-                game.Run(60.0);
-            }
+            using (var window = new Window(800, 600, "LearnOpenTK - Transformations"))
+                window.Run(60.0);
```
Comment (Perksey) [inline Chapter 1/6 - Transformations/Program.cs]: Revert.

File: Chapter 1/7 - Coordinates Systems/Program.cs [hunk 10]
```diff
@@ -1,13 +1,11 @@
...
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            using (Game game = new Game(800, 600, "LearnOpenTK - Coordinates Systems"))
-            {
-                game.Run(60.0);
-            }
+            using (var window = new Window(800, 600, "LearnOpenTK - Coordinates Systems"))
+                window.Run(60.0);
```
Comment (Perksey) [inline Chapter 1/7 - Coordinates Systems/Program.cs]: Revert.

File: Chapter 1/8 - Camera/Program.cs [hunk 11]
```diff
@@ -1,13 +1,11 @@
...
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            using (Game game = new Game(800, 600, "LearnOpenTK - Camera Tutorial"))
-            {
-                game.Run(60.0);
-            }
+            using (var window = new Window(800, 600, "LearnOpenTK - Camera"))
+                window.Run(60.0);
```
Comment (Perksey) [inline Chapter 1/8 - Camera/Program.cs]: Revert.

File: Common/Shader.cs [hunk 12]
```diff
@@ -7,122 +7,124 @@
...
-            string readContents;
-
-            using (StreamReader streamReader = new StreamReader(path, Encoding.UTF8))
-            {
-                readContents = streamReader.ReadToEnd();
-            }
-
-            return readContents;
+            using (var sr = new StreamReader(path, Encoding.UTF8))
+                return sr.ReadToEnd();
```
Comment (ITALIA195) [inline Common/Shader.cs]: Fixed missing braces in last commit

File: Chapter 1/8 - Camera/Program.cs (hunk 11 above)
Comment (ITALIA195) [reply Chapter 1/8 - Camera/Program.cs]: Fixed

File: Chapter 1/7 - Coordinates Systems/Program.cs (hunk 10 above)
Comment (ITALIA195) [reply Chapter 1/7 - Coordinates Systems/Program.cs]: Fixed

File: Chapter 1/6 - Transformations/Program.cs (hunk 9 above)
Comment (ITALIA195) [reply Chapter 1/6 - Transformations/Program.cs]: Fixed

File: Chapter 1/5 - Multiple Textures/Program.cs (hunk 8 above)
Comment (ITALIA195) [reply Chapter 1/5 - Multiple Textures/Program.cs]: Fixed

File: Chapter 1/4 - Textures/Shaders/shader.vert (hunk 7 above)
Comment (ITALIA195) [reply Chapter 1/4 - Textures/Shaders/shader.vert]: Fixed

File: Chapter 1/4 - Textures/Program.cs (hunk 5 above)
Comment (ITALIA195) [reply Chapter 1/4 - Textures/Program.cs]: Fixed

File: Chapter 1/3 - Element Buffer Objects/Program.cs (hunk 4 above)
Comment (ITALIA195) [reply Chapter 1/3 - Element Buffer Objects/Program.cs]: Fixed

File: Chapter 1/2 - Hello Triangle/Program.cs (hunk 3 above)
Comment (ITALIA195) [reply Chapter 1/2 - Hello Triangle/Program.cs]: Fixed

File: Chapter 1/1 - Creating a Window/Program.cs (hunk 2 above)
Comment (ITALIA195) [reply Chapter 1/1 - Creating a Window/Program.cs]: Fixed

File: Chapter 1/4 - Textures/Shaders/shader.frag (hunk 6 above)
Comment (ITALIA195) [reply Chapter 1/4 - Textures/Shaders/shader.frag]: Fixed

File: Chapter 1/4 - Textures/Shaders/shader.vert (hunk 7 above)
Comment (ITALIA195) [reply Chapter 1/4 - Textures/Shaders/shader.vert]: Fixed

**Comment by devvoid** [comment]
It all looks good to me. Thanks for your contribution!

**Pull request merged**